public/editor/scripts/project-list.js

```diff
@@ -81,19 +81,19 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
 
   $("#project-list").prepend(favoriteProjectsElementList);
 
-  $(".project-delete").click(function() {
-    // TODO: we can do better than this, but let's at least make it harder to lose data.
-    if(!window.confirm("{{ deleteProjectConfirmationText }}")) {
-      return false;
-    }
+  var projectsToDelete = [];
 
-    var project = $(this).closest(".project");
+  function deleteProject(project){
     var projectId = project.attr("data-project-id");
     var projectElementId = project.attr("id");
     $("#" + projectElementId + " > .project-title").off("click");
 
-    analytics.event({ category : analytics.eventCategories.PROJECT_ACTIONS, action : "Delete Project" });
```

Comment:
Woops I think i might have messed this up when I was trying to rebase my branch.